membership-common/src/main/scala/com/gu/memsub/subsv2/reads/CommonReads.scala

```diff
@@ -47,37 +28,8 @@ object CommonReads {
     }
   }
 
-  implicit class FailureAggregatingOrElse[X](validationNel: ValidationNel[String, X]) {
-    def orElse2[A >: X, Y <: A](fallback: ValidationNel[String, Y]): ValidationNel[String, A] = {
-      validationNel match {
-        case Failure(firstFailures) =>
-          fallback
-            .leftMap(moreFailures => NonEmptyList.fromSeq[String](firstFailures.head, firstFailures.tail.toList ++ moreFailures.list.toList))
-            .map(identity[A])
-        case Success(_) => validationNel.map(identity[A])
-      }
-    }
-  }
-
-  implicit val pricingSummaryReads = new Reads[PricingSummary] {
-    override def reads(json: JsValue): JsResult[PricingSummary] = {
-
-      // for subscriptions our pricing summary is a string i.e. 10GBP, for the catalog its an array
-      val normalisedPricingList = json.validate[List[String]] orElse json.validate[String].map(List(_))
-
-      val parsedPrices = normalisedPricingList.flatMap { priceStrings =>
-        priceStrings
-          .map(PriceParser.parse)
-          .sequence[Option, Price]
-          .toJsSuccess(s"Failed to parse $normalisedPricingList")
-      }
-
-      parsedPrices.map(priceList => priceList.map(p => p.currency -> p).toMap).map(PricingSummary(_))
-    }
-  }
```

Comment:
moved to SubJsonReads below as we only read the pricing summary from a subscription, not the catalog any more